src/components/Editor/DrawingBoard/Canvas/Worker/EraserWorker.ts

```diff
@@ -0,0 +1,99 @@
+import { TimeTicket } from 'yorkie-js-sdk';
+import { Root, Point } from 'features/docSlices';
+import { ToolType } from 'features/boardSlices';
+import { createEraserLine, fixEraserPoint } from '../line';
+import Worker from './Worker';
+import { compressPoints, checkLineIntersection, isInnerBox, isSelectable } from '../utils';
+import * as scheduler from '../scheduler';
+
+class EraserWorker implements Worker {
+  type = ToolType.Eraser;
+
+  private update: Function;
+
+  private emit: Function;
```

Comment:
I think `emit ` is EventDispatcher's emit function. I'm thinking about if there is a good way to draw the shape(`this.emit('renderAll', root.shapes);`) rather than we pass emit function when we use EraserWorker. 🤔